src/BasisTheory.net/Tokens/TokenClient.cs

```diff
@@ -309,7 +326,35 @@ public Token Create(TokenCreateRequest token, RequestOptions requestOptions = nu
 
             return await PostAsync<Token>(BasePath, token, requestOptions, cancellationToken);
         }
+        
+        public Token Update(Guid tokenId, TokenUpdateRequest updateRequest, RequestOptions requestOptions = null)
+        {
+            return Patch<Token>($"{BasePath}/{tokenId}", updateRequest, requestOptions);
+        }
+        
+        public Token Update(string tokenId, TokenUpdateRequest updateRequest, RequestOptions requestOptions = null)
+        {
+            return Patch<Token>($"{BasePath}/{tokenId}", updateRequest, requestOptions);
+        }
+        
+        public async Task<Token> UpdateAsync(
+            Guid tokenId,
+            TokenUpdateRequest updateRequest,
+            RequestOptions requestOptions = null,
+            CancellationToken cancellationToken = default)
+        {
+            return await PatchAsync<Token>($"{BasePath}/{tokenId}", updateRequest, requestOptions, cancellationToken);
+        }
 
+        public async Task<Token> UpdateAsync(
+            string tokenId,
+            TokenUpdateRequest updateRequest,
+            RequestOptions requestOptions = null,
+            CancellationToken cancellationToken = default)
+        {
+            return await PatchAsync<Token>($"{BasePath}/{tokenId}", updateRequest, requestOptions, cancellationToken);
+        }
+        
```

Comment:
This endpoint is now expecting a `Content-Type` header of `application/merge-patch+json`. Can we set that and add tests that the correct content-type is sent on this request?